wirecard/oxid-ee
# WDPD-109 Create hierarchical transaction overview

### This PR

* Adds a tab to the order overview to display related transactions
* The related transactions are displayed hierarchically and sorted by time of execution
* Refactors admin tabs
* Adds a TransactionList model to get lists of transactions
* Fixes a Javascript conflict in the payment selection
* Adds a unit test case extension to be able to run integration tests more easily

### TODO

* Unit tests

### How to Test

* Go to Administer _Orders → Orders_ and click on an order that has been made with a Wirecard payment
* Click the _Transactions_ tab and confirm that:
  * All transactions related to this order are displayed
  * Child transactions are indented to show the hierarchy
  * Transaction in one level are sorted by date

### Jira Links

* WDPD-109: https://jira.parkside.at/browse/WDPD-109
